test/sqlsrv/sqlsrv_simple_fetch_variants_.phpt

```diff
@@ -0,0 +1,194 @@
+--TEST--
+Test simple insert and fetch sql_variants as strings using inputs of various data types
```

Comment:
This test looks so much similar to sqlsrv/sqlsrv_simple_fetch_variants.phpt? What's the difference?